File: Assignment_5/problem2.c
/////////////////////////////////////////////////////////////////
//
// Name :          program2.c
// Discription :   Display numbers of input
// Input :         integer
// Output :        integer
// Author :        Shivam Rajendra Kale
// Date :          14/05/2025
//
/////////////////////////////////////////////////////////////////

/*
    Start
        Accept Number from User
            Display one by numbers 
        
    Stop         
*/

#include<stdio.h>

void Display(int iNo)
{
    int iCnt = 0;

    for(iCnt = 1; iCnt <= iNo; iCnt++)
    {
        printf("%d\t",iCnt);
    }

}

int main()
{
    int iValue = 0;

    printf("Enter number : ");
    scanf("%d",&iValue);

    Display(iValue);

    return 0;
}

File: Assignment_java_2/problem2.java
/////////////////////////////////////////////////////////////////
//
// Name :          program2.java
// Discription :   Display numbers of * on screen
// Input :         integer
// Output :        String
// Author :        Shivam Rajendra Kale
// Date :          06/06/2025
//
/////////////////////////////////////////////////////////////////

/*
    Start
        Accept Number from User
          Display number of * on screen
    Stop         
*/

import java.util.Scanner;

class Display
{
    public int iNo1;

    public Display(int iNo1)
    {
        this.iNo1 = iNo1;
    }

    public void DisplayX()
    {
        if(iNo1 < 0)    //Updator
        {
            iNo1 = -iNo1;
        }

        while(iNo1 >= 1)
        {
            System.out.println("*");
            iNo1--;
        }
    }
}
class problem2
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter number :");
        int iValue = sobj.nextInt();

        Display dobj = new Display(iValue);

        dobj.DisplayX();
    }

}


